HLTrigger/Configuration/python/customizeHLTForPFTrackingPhaseI2017.py

```diff
@@ -0,0 +1,294 @@
+import FWCore.ParameterSet.Config as cms
+
+
+def producers_by_type(process, *types):
+    return (module for module in process._Process__producers.values() if module._TypedParameterizable__type in types)
```

Comment:
And a more general comment. Could we have some central place to to put all these nice helpers instead of copy-pasting them around?

@Dr15Jones